lang/admin/hi.json

```diff
@@ -0,0 +1,5 @@
+{
+    "en": {
```

Comment:
Please change this key to "hi" instead of "en", as it corresponds to the language identifier